WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModelBase.cs

```diff
@@ -27,12 +27,11 @@ public WalletViewModelBase(Wallet wallet)
 			Wallet = wallet;
 			Title = WalletName;
 
-			WalletState = wallet.State;
-
 			Observable.FromEventPattern<WalletState>(wallet, nameof(wallet.StateChanged))
 				.ObserveOn(RxApp.MainThreadScheduler)
 				.Subscribe(x => WalletState = x.EventArgs)
 				.DisposeWith(_disposables);
+			WalletState = wallet.State;
```

Comment:
interested what changing the order here does?

Reply:
I tried there something and left it like that. Reverted.